src/storage/hummock_sdk/src/lib.rs

```diff
@@ -355,3 +355,25 @@ impl EpochWithGap {
         self.0 & EPOCH_SPILL_TIME_MASK
     }
 }
+
+pub trait ProtoSerializeSizeEstimatedExt {
```

Comment:
Why are the traits all named with the suffix 'Ext'? Should they give default implementation and implements for all types that applies the trait without `Ext`?